### ISSUE TITLE
[feature request] Ability to disable block swapping when a block is placed by the player

### ISSUE THREAD

**Issue by rosy612** [CorgiTaco-MC/Block-Swap#10]
idk if this is already possible but it would be awesome to be able to disable the feature where the blocks are swapped when placed by the player. for example, if you wanted to remove hay bales from villages you would do "minecraft:hay_block" : "minecraft:air" in the config, but now you cant place hay bales. This mod is awesome but this is something I would realllllly like to see :)

**Comment by blockgoblin31** [comment]
me as well

**Comment by K4N0** [comment]
Yes please. I would like to replace some blocks, but only upon world gen, Not when a player places a block.

**Comment by K4N0** [comment]
I guess if you pregenerate a world, you can then remove the mod.

**Comment by blockgoblin31** [comment]
I'm asking as a pack dev, so that sorta isn't an option. also what about new chunks?

**Comment by NiveStinks** [comment]
This is the feature I want most. It'd be especially useful once Trial Ruins are added in 1.21 so you can swap out the copper blocks to something that can't be reverse crafted back into copper, for example.
